### ISSUE TITLE
Passing empty strings to ordered_placement_strategy blocks causes panic.

### ISSUE THREAD

**Issue by breathingdust** [terraform-providers/terraform-provider-aws#13146]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* aws_ecs_service

### Panic Output

```
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: panic: interface conversion: interface {} is nil, not map[string]interface {}
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: goroutine 72 [running]:
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: github.com/terraform-providers/terraform-provider-aws/aws.expandPlacementStrategy(0xc000a8a4e0, 0x3, 0x3, 0x4761800, 0xc000a3f6a0, 0x1, 0xc000a820d0, 0x1)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_ecs_service.go:822 +0x43d
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: github.com/terraform-providers/terraform-provider-aws/aws.resourceAwsEcsServiceCreate(0xc000890690, 0x4f00f20, 0xc0004faa00, 0x2, 0x969c460)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_ecs_service.go:460 +0x8b1
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc0005b62d0, 0xc000c226e0, 0xc0009518a0, 0x4f00f20, 0xc0004faa00, 0xc000715101, 0xc000ab0120, 0xc0007151a0)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/schema/resource.go:310 +0x36c
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc000134b80, 0xc000d7f9c8, 0xc000c226e0, 0xc0009518a0, 0xc000715198, 0xc000006e48, 0x4d17c40)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/schema/provider.go:294 +0xa0
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc000006b08, 0x6396960, 0xc0007f7c80, 0xc0007e55e0, 0xc000006b08, 0xc0007f7c80, 0xc000d93b30)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin/grpc_provider.go:885 +0x8bb
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x587f680, 0xc000006b08, 0x6396960, 0xc0007f7c80, 0xc000ba44e0, 0x0, 0x6396960, 0xc0007f7c80, 0xc00090d200, 0x83f)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5/tfplugin5.pb.go:3305 +0x21e
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: google.golang.org/grpc.(*Server).processUnaryRPC(0xc000059e00, 0x63b8780, 0xc000e9a180, 0xc000bc4e00, 0xc0007a9e60, 0x966d3c0, 0x0, 0x0, 0x0)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:1024 +0x4fb
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: google.golang.org/grpc.(*Server).handleStream(0xc000059e00, 0x63b8780, 0xc000e9a180, 0xc000bc4e00, 0x0)
2020-04-23T06:32:35.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:1313 +0xd9e
2020-04-23T06:32:35.317-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc00003a0e0, 0xc000059e00, 0x63b8780, 0xc000e9a180, 0xc000bc4e00)
2020-04-23T06:32:35.317-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:722 +0xc2
2020-04-23T06:32:35.317-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: created by google.golang.org/grpc.(*Server).serveStreams.func1
2020-04-23T06:32:35.317-0400 [DEBUG] plugin.terraform-provider-aws_v2.58.0_x4.exe: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:720 +0xa8
2020/04/23 06:32:35 [DEBUG] aws_ecs_service.ecs-service: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2020/04/23 06:32:35 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
```

### Steps to Reproduce

1. `terraform apply`

**Comment by bflad** [comment]
Fix submitted (with bonus plan-time validation): https://github.com/terraform-providers/terraform-provider-aws/pull/13926

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.68.0 of the Terraform AWS Provider, likely tomorrow. 👍
